research-2023/effort-mapping-model/2023-04-28.001.md

```diff
@@ -0,0 +1,321 @@
+
+Very rough concept:
+```xml
+<qualifier
+	subject=“source|target|both”
+	has=“requirement|incompatibility”
+	category=“restricted|addressable|blocked”>
+    Description of qualifier...
+</qualifier>
+```
+
+---
+
+### Option: String Marker in Qualifer
+
+Additional attributes could be considered for marking a start and end to the control/statement string to highlight the portion referenced by the qualifier.
+
+```
+	start=“starting position in string”
+	end=“ending position in string”
+```
+
+There are other potential approaches here, and this is likely an "overdone" aspect of our desire to implement just enough to make the model useful as a first release.
+
+---
+
+### New Assembly: Provenance
+
+This assembly applies to the entire mapping and provides clear context on intended use, the author(s) of the mapping, the manner it was achieved, and the state of the mapping.  If we imagine a repository of 200 mapping documents, this provides enough context to traverse all mappings, or query them, in specific ways to eliminate undesired mappings depending on the use case.  For example, one may wish to only use content produced by people, with a complete status.  Another example might allow a CISO to visually explore future or proposed changes to a standard.
+
+Very rough concept:
+```xml
+<provenance
+	responsible-parties=""
+	method=“human|automation”
+	matching=“lexical|logical|semantic|etc”
+	confidence-score=""
+	status=“complete|not-complete|draft|deprecated|superseded|etc”>
+    Description of context and intended use...
+</provenance>
+```
+
+---
+
+### New Relationship: Not Mapped
+
+I incorrectly interpreted the addition of an `unspecified` relationship which would allow the addition of any unmapped control as sufficient, and this needs a quick spiral.  The cardinality requires at least one source and one target, so this will not work in all cases.  Michaela recommended a "mini-profile" to capture this information in the model, and I did not have time to pull this part together.
+
+---
+
+## Recommended Actions
+
+- Prepare a feature request to add a qualifier assembly/tag as a part of the map assembly to describe requirements, incompatibilities and gaps that are identified between a target and source.
+- Prepare a feature request to add a required provenance assembly to document contextual information and responsibility for the mapping.
+- Determine a path forward for a section of the model to specify a list of unmapped controls for source and target.
+- Consider evidence as an important dimension of equivalency in some contexts, particularly if an organization is using a mapping to prepare for meeting a new standard based on another framework.  If this particular approach requires a more in-depth synthesis using profiles and SSP documents, we should produce a guide for this.
```

Comment:
Minor and non-blocking: while I work on more detailed feedback, can you convert these into a numbered list and not bulleted? It would be great if we can specifically address final decision feedback item by item when it comes to that next week. Thanks.

```suggestion
1. Prepare a feature request to add a qualifier assembly/tag as a part of the map assembly to describe requirements, incompatibilities and gaps that are identified between a target and source.
1. Prepare a feature request to add a required provenance assembly to document contextual information and responsibility for the mapping.
1. Determine a path forward for a section of the model to specify a list of unmapped controls for source and target.
1. Consider evidence as an important dimension of equivalency in some contexts, particularly if an organization is using a mapping to prepare for meeting a new standard based on another framework.  If this particular approach requires a more in-depth synthesis using profiles and SSP documents, we should produce a guide for this.
```